### DISCUSSION TITLE
How to use external migration tools with Diesel

### DISCUSSION THREAD

**Discussion by RustyNova016** [diesel-rs/diesel#3916] [Q&A]
Here's my current setup. I got a web app running on node.js, and using Drizzle ORM for the database
I am making a rust program that need to use this database, and I am interested in using diesel for it.

I plan to keep the migration tools on DrizzleORM, as it's just easier than manually doing SQL files, and better to break less stuff on the javascript side.
But now I need a way to migrate Diesel's schema when updated. I know I can update the models directly through Diesel CLI Extension, but is there a way to update shema.rs directly from infering the database?
The database will always be active at compile time, and I can modify my migration script on drizzle's side to execute other CLI commands.

Any help would be great.

**Comment by weiznich** [comment]
You can use the `print-schema` subcommand of diesel-cli for that. See the `--help` output for relevant options.
